uaclient/tests/test_security.py

```diff
@@ -1762,13 +1949,13 @@ def test_messages_for_affected_packages_when_service_kept_disabled(
             "",
         )
         type(m_entitlement_obj).name = mock.PropertyMock(
-            return_value="esm-apps"
+            return_value="esm-infra"
         )
 
         cfg = FakeConfig()
         cfg.for_attached_machine()
         with mock.patch.object(
-            sec, "ENTITLEMENT_CLASS_BY_NAME", {"esm-apps": m_entitlement_cls}
+            sec, "ENTITLEMENT_CLASS_BY_NAME", {"esm-infra": m_entitlement_cls}
```

Comment:
I think I understand the new tests, but can you explain why we changed several existing tests to reference "infra" instead of "apps" like right here?

Reply:
My rationale here is just to make the test seems more "right". I mean that we expect `esm-infra`  package related fixes to work fine, which will not happen to `esm-apps` by default now. So I am just changing the name to reflect that.

But feature wise, it will not change anything, since we are mocking the service here, as you have noted.

Reply:
Ahhh I understand - it would be confusing for the tests to expect output that doesn't align with the defaults in actual usage. Yeah I think that's a good idea!